pkg/exchange/binance/parse.go

```diff
@@ -462,6 +467,72 @@ func parseDepthEvent(val *fastjson.Value) (*DepthEvent, error) {
 	return depth, err
 }
 
+type MarketTradeEvent struct {
+	EventBase
+	Symbol   string           `json:"s"`
+	Quantity fixedpoint.Value `json:"q"`
+	Price    fixedpoint.Value `json:"p"`
+
+	BuyerOrderId  int64 `json:"b"`
+	SellerOrderId int64 `json:"a"`
+
+	OrderTradeTime int64 `json:"T"`
+	TradeId        int64 `json:"t"`
+
+	IsMaker bool `json:"m"`
```

Comment:
you need to define the upper case `M` to fix the parsing.

Reply:
done